README.md

```diff
@@ -1,4 +1,4 @@
-# Getting Started with Create React App
+# Getting Started with Create React App 
```

Comment:
One space missed! Add some more spaces!!!

Reply:
Added spaces. Sorry for such mistake.